ldap/admin/src/logconv.pl

```diff
@@ -2852,12 +2852,13 @@ sub parseLineNormal
 						 $stats->{'unbind'},
 						 $stats->{'notesA'},
 						 $stats->{'notesU'},
+						 $stats->{'notesF'},
 						 $stats->{'etime'}),
 					"\n" );
 			} else {
 				$stats->{'fh'}->print(
 					"Time,time_t,Results,Search,Add,Mod,Modrdn,Moddn,Compare,Delete,Abandon,".
-					"Connections,SSL Conns,Bind,Anon Bind,Unbind,Unindexed search,Unindexed component,ElapsedTime\n"
+					"Connections,SSL Conns,Bind,Anon Bind,Unbind,Unindexed search,Unindexed component,Invalid filter,ElapsedTime\n"
```

Comment:
I do not think that "Invalid" is the right term .I would rather say "Inefficient"

Reply:
This term is documented:
https://www.port389.org/docs/389ds/design/filter-syntax-verification.html

https://access.redhat.com/documentation/en-us/red_hat_directory_server/12/html/configuration_and_schema_reference/assembly_core-server-configuration-attributes_config-schema-reference-title#ref_nsslapd-verify-filter-schema_assembly_cn-config

So I'd rather stick with it.